client/selectors/events.ts

```diff
@@ -1,6 +1,8 @@
 import {createSelector} from 'reselect';
 import {get, sortBy} from 'lodash';
 
+// PR-TODO: look again at diff of this file
```

Comment:
Was this self review completed?

Reply:
I forgot to finalize this one. I was planning to simply add a tag comment here, but I found the selector was being used by component that is itself unused so I was able to drop the selector too.